src/provisioner/provisioner.py

```diff
@@ -226,9 +220,14 @@ def _provision_components(cortx_conf: MappedConf, interfaces: Enum, apply_phase:
                         continue
                 CortxProvisioner._update_provisioning_status(
                         cortx_conf, node_id, apply_phase, ProvisionerStatus.PROGRESS.value)
-                cmd = (
-                    f"/opt/seagate/cortx/{component_name}/bin/{component_name}_setup {interface.value}"
-                    f" --config {cortx_conf._conf_url} --services {service}")
+                if interface.value == 'upgrade':
+                    cmd = (
+                        f"/opt/seagate/cortx/{component_name}/bin/{component_name}_setup {interface.value}"
+                        f" --delta {const.DELTA_PATH} --config {cortx_conf._conf_url} --services {service}")
```

Comment:
Please add a TODO note here that says  remove --config parameter once components are ready to accept delta only conf for upgrade.

Reply:
done